tests/test_extentions.py

```diff
@@ -0,0 +1,16 @@
+import spacy
+from spacy.tokens import Doc, Span, Token
+from relationextraction import SpacyRelationExtractor
+from collections import Counter
+from heads_extract_component import HeadwordsExtraction
+
+
+def test_extentions():
+
+    nlp = spacy.load("en_core_web_lg")
+    doc = nlp("Mette Frederiksen is the Danish Politician")
+    nlp.add_pipe("heads_extraction")
+
+    assert isinstance(doc[:]._.most_common_ancestor, Span)  # Doc
```

Comment:
```suggestion
    assert isinstance(doc._.most_common_ancestor, Span)  # Doc
```
I assume this is what you want to test?